charts/nextcloud/templates/_helpers.tpl

```diff
@@ -151,7 +151,11 @@ Create environment variables used to configure the nextcloud container as well a
       name: {{ .Values.nextcloud.existingSecret.secretName | default (include "nextcloud.fullname" .) }}
       key: {{ .Values.nextcloud.existingSecret.passwordKey | default "nextcloud-password" }}
 - name: NEXTCLOUD_TRUSTED_DOMAINS
+{{- if .Values.metrics.enabled }}
+  value: {{ .Values.nextcloud.host }} metrics-server
```

Comment:
Does this space separation work?

Reply:
Yes it does:
```
        - name: NEXTCLOUD_TRUSTED_DOMAINS
          value: nextcloud.mediaserver.fritz.box metrics-server
```

Because Kubernetes assumes it's a string. But I can add double quotes for safety.

Reply:
I'd prefer that